daemon/cmd/daemon_test.go

```diff
@@ -158,13 +151,8 @@ func (ds *DaemonSuite) SetUpTest(c *C) {
 				cs.Disable()
 				return cs
 			},
-			func() datapath.Datapath { return fakeDatapath.NewDatapath() },
-			func(dp datapath.Datapath) datapath.NodeIDHandler { return dp.NodeIDs() },
 			func() *option.DaemonConfig { return option.Config },
-			func() cnicell.CNIConfigManager { return &fakecni.FakeCNIConfigManager{} },
-			func() signalmap.Map { return fakesignalmap.NewFakeSignalMap([][]byte{}, time.Second) },
-			func() authmap.Map { return fakeauthmap.NewFakeAuthMap() },
-			func() egressmap.PolicyMap { return nil },
+			fakeDatapath.Cell,
```

Comment:
If `fakeDatapath` provides the `tables` cell then can we remove `tables.Cell` below?

Reply:
Good catch, yes.